osbs/schemas/container.json

```diff
@@ -106,6 +106,24 @@
                         "enable-party-popper"
                       ]
                   }
+                },
+                "packages": {
+                  "description": "the packages object to pass to the Cachito request",
+                  "type": ["object", "null"],
+                  "examples": [
+                    {"npm": [{"path": "client"}, {"path": "proxy"}]}
+                  ],
+                  "additionalProperties" : {
+                    "type" : "array",
+                    "items": {
+                      "type": "object",
```

Comment:
is this object properties restricted to `path`? If so, can we make it explicit here?

Note that this file is linked in the upstream documentation (in hopes that users will refer to it when writing their container.yaml files; some do).

Reply:
It is as of now, but it could be expanded as use-cases pop up. I specifically made this less restrictive so that the schema does not need to be updated every time Cachito adds support for a new property.